### DISCUSSION TITLE
NextJS - Critical dependency: the request of a dependency is an expression

### DISCUSSION THREAD

**Discussion by h36ahmed** [google/model-viewer#4512] [Q&A]
### Description

web:dev:  ⚠ ../../node_modules/.pnpm/@google+model-viewer@3.3.0_three@0.157.0/node_modules/@google/model-viewer/dist/model-viewer.min.js
web:dev: Critical dependency: the request of a dependency is an expression

I have added three package in my package.json and still keeps complaining

- How do you solve this issue? I am using Next 13 and it's a warning not an error.

### Version

- model-viewer: v3.3.0

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [x] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Sounds like a NextJS issue? Was it working before with v3.2?

**Comment by h36ahmed** [comment]
Even with v3.2, it was giving me the same issue

**Reply by elalish** [reply]
Have you searched our discussions for NextJS? You may find some helpful tidbits. Probably just a configuration thing.

**Comment by h36ahmed** [comment]
Perhaps this may be PNPM thing `../../node_modules/.pnpm/@google+model-viewer@3.3.0_three@0.157.0/node_modules/@google/model-viewer/dist/model-viewer.min.js`
